package.json

```diff
@@ -76,13 +74,8 @@
     "dirty-chai": "^2.0.1",
     "documentation": "^13.0.1",
     "electron-mocha": "^8.2.0",
-    "eslint": "^6.3.0",
-    "eslint-config-standard": "^14.1.1",
-    "eslint-plugin-import": "^2.21.2",
-    "eslint-plugin-no-only-tests": "^2.4.0",
-    "eslint-plugin-node": "^11.0.0",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-standard": "^4.0.1",
+    "eslint": "^6.8.0",
+    "eslint-config-ipfs": "git://github.com/gozala/eslint-config-ipfs#default",
```

Comment:
Please publish this as a module & swap out the gh url before merging.

Reply:
@achingbrain of course, this was just for illustration purposes

Reply:
```suggestion
    "eslint-config-ipfs": "^0.1.0",
```